articles/tokens/access-token.md

```diff
@@ -0,0 +1,142 @@
+
+Also, the consent dialog might be displayed again if the access level changes. For example, if the user has granted read access but the functionality changes so write access is required as well, the user will have to use the consent dialog to grant the additional access.
+
+In some cases, consent can also be pre-configured administratively. This typically occurs when the user is an employee, and there are a set of company-specific applications that are always allowed access to employee data.
+
+### Server-to-server interactions
+
+Access tokens can also be issued directly to applications, permitting access to APIs that don't provide user-specific data. Such scenarios involve server-to-server interactions.
+
+For example, a reverse geocoding API that accepts a latitude, longitude coordinate and returns a readable place name does not access user-owned data. In such cases a backend server needs to call the geocoding API in order to perform the translation.
+
+Server-to-server access tokens can be obtained using the `client_credentials` flow, exchanging the client's credentials for an API access token.
+
+```
+POST /oauth/token
+Host​: {tenant}.auth0.com
+Content-Type: application/x-www-form-urlencoded
+
+grant_type=client_credentials&
+client_id={CLIENT_ID}&
+client_secret={CLIENT_SECRET}
+audience=https://api.example.com/geocoding/v1/
+```
+
+The result will be an access token that can be used to make requests to the geocoding API.
+
+```
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "access_token":"ey...",
+  "token_type":"Bearer",
+  "expires_in":86400,
+  "example_parameter":"example_value"
+}
+```
+
+In order to obtain this access token, the client must first have been granted permission to access the geocoding API. This is typically done by requesting access from the administrator of the geocoding API.
+
+## Authorize access tokens
+
+Once a client has obtained an access token, it will include that token as a credential when making API requests.
+
+```
+GET /calandar/v1/events
+Host​: api.example.com
+
+Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJodHRwczovL2V4YW1wbGUuYXV0aDAuY29tLyIsImF1ZCI6Imh0dHBzOi8vYXBpLmV4YW1wbGUuY29tL2NhbGFuZGFyL3YxLyIsInN1YiI6InVzcl8xMjMiLCJpYXQiOjE0NTg3ODU3OTYsImV4cCI6MTQ1ODg3MjE5Nn0.CA7eaHjIHz5NxeIJoFK9krqaeZrPLwmMmgI_XiQiIkQ
+```
+
+The token in this example decodes to the following claims:
+
+```json
+{
+  "alg": "HS256",
+  "typ": "JWT"
+}
+.
+{
+  "iss": "https://example.auth0.com/",
+  "aud": "https://api.example.com/calandar/v1/",
+  "sub": "usr_123",
+  "scope": "read write",
+  "iat": 1458785796,
+  "exp": 1458872196
+}
+```
+
+Before permitting access to the API using this token, the API must do the following:
```

Comment:
The API also needs to validate that the incoming token has the correct scopes to perform the requested operation.